lumicks/pylake/detail/tests/test_bead_crop.py

```diff
@@ -5,7 +5,7 @@
 import pytest
 import matplotlib.pyplot as plt
 
-from lumicks.pylake.detail.bead_cropping import find_beads
+from lumicks.pylake.detail.bead_cropping import find_beads, find_beads_template
```

Comment:
since we have 2, maybe we should have a more distinct name for `find_beads`? `find_beads_brightness`?

Reply:
yeah, good call